test/test_dynamic_transform.cpp

```diff
@@ -618,4 +618,107 @@ TEST_F(NVFuserTest, DynamicTransform11_CUDA) {
       info2.toString());
 }
 
+// Test FusionExecutorCache with dynamic reshapes
+TEST_F(NVFuserTest, DynamicTransformFusionExecutorCache_CUDA) {
```

Comment:
I think it's also a good idea to have a shmoo test just to make sure the whole process of concretization and caching works as intended with more complex problem sizes. There's shmoo tests in test_gpu_view.cpp. Maybe we should create a dynamic version of some of the shmoo tests. Note they are slow, so we also don't want too many shmoo cases.